src/redisearch.h

```diff
@@ -51,6 +51,8 @@ typedef enum {
 #define isSpecHash(spec) (spec->rule && spec->rule->type == DocumentType_Hash)
 #define isSpecJson(spec) (spec->rule && spec->rule->type == DocumentType_Json)
 
+#define isMockRedis (!RedisModule_CreateTimer)
```

Comment:
The `isMockRedis` (maybe `isRedisMock` is a better name) is a good idea, the implementation is shaky. Like, we need to remember that CreateTimer is an indication of mock - maybe we can come up with something more robust? If not, let's at least add a comment.

Reply:
Change to `RS_IsMock`